pandas/io/parsers.py

```diff
@@ -116,13 +116,16 @@
     Values to consider as True
 false_values : list, default None
     Values to consider as False
-skipinitialspace : boolean, default False
+skip_initial_space : boolean, default False
     Skip spaces after delimiter.
-skiprows : list-like or integer, default None
+skip_rows : list-like or integer, default None
     Line numbers to skip (0-indexed) or number of lines to skip (int)
     at the start of the file
-skipfooter : int, default 0
+skip_footer : int, default 0
     Number of lines at bottom of file to skip (Unsupported with engine='c')
+skipfooter : int, default None
+    DEPRECATED: this argument will be removed in a future version. Please
+    use the `skip_footer` parameter instead.
```

Comment:
Same comment here: why add it for skipfooter?

In the docstring, I would add a similar note about other keyword arguments without underscores being accepted (can be the same as in io.rst) in the 'Notes' section.